### PR TITLE
feat: attach '(latest)' if version is latest in CollectorBaseInfoSection

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

Please review in order by commits.

https://github.com/cloudforet-io/console/assets/26986739/1d6ba647-6459-4c95-b2f2-12506de5a2e0

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorVersionForm.vue [hunk 1]
```diff
@@ -33,23 +33,26 @@ import {
...
+
 const emit = defineEmits<{(event: 'update:isVersionValid', value: boolean): void;
 }>();
 
 const state = reactive({
     pluginId: computed<string|undefined>(() => collectorFormStore.pluginId),
-    versions: [] as MenuItem[],
+    versionItems: computed<MenuItem[]>(() => collectorFormState.versions.map((value, index) => {
+        if (index === 0) return { type: 'item', label: `${value} (latest)`, name: value };
+        return { type: 'item', label: value, name: value };
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorVersionForm.vue:54]: question: Is `versions` always sorted? 🧐

**Review by yuda110 — APPROVED** [review]
LGTM!!

File: apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorVersionForm.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorVersionForm.vue:54]: yes!

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue [hunk 2]
```diff
@@ -106,17 +106,17 @@ const collectorFormStore = useCollectorFormStore();
...
 const state = reactive({
+    fields: computed<DefinitionField[]>(() => [
+        { name: 'pluginName', label: i18n.t('INVENTORY.COLLECTOR.DETAIL.PLUGIN') },
+        { name: 'plugin_info.plugin_id', label: i18n.t('INVENTORY.COLLECTOR.DETAIL.PLUGIN_ID') },
+        { name: 'plugin_info.version', label: i18n.t('INVENTORY.COLLECTOR.DETAIL.VERSION'), disableCopy: state.isCollectorAutoUpgrade },
+        { name: 'tags', label: i18n.t('INVENTORY.COLLECTOR.DETAIL.TAG') },
+        { name: 'plugin_info.upgrade_mode', label: i18n.t('INVENTORY.COLLECTOR.DETAIL.AUTO_UPGRADE'), disableCopy: true },
+        { name: 'last_collected_at', label: i18n.t('INVENTORY.COLLECTOR.DETAIL.LAST_COLLECTED') },
+        { name: 'created_at', label: i18n.t('INVENTORY.COLLECTOR.DETAIL.CREATED') },
+    ]),
```
Comment (skdud4659) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue:119]: lovit!

**Review by skdud4659 — APPROVED** [review]
LGTM!!

**Pull request merged**